Mage/src/main/java/mage/filter/common/FilterCreatureOrPlaneswalkerCard.java

```diff
@@ -0,0 +1,30 @@
+
+
+package mage.filter.common;
+
+import mage.constants.CardType;
+import mage.filter.FilterCard;
+import mage.filter.predicate.Predicates;
+
+public class FilterCreatureOrPlaneswalkerCard extends FilterCard {
```

Comment:
you don't need to make a whole new class for this, just make a `FilterCard` and add the predicates to it